proto/server/main.cc

```diff
@@ -0,0 +1,8 @@
+#include "server.h"
+
+#include <string>
+
+int main(){
+  RunServer("0.0.0.0", "1234");
```

Comment:
Pass this in as a flag so that people can change it at runtime.

Reply:
Done